index.html

```diff
@@ -2360,9 +2358,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-related-head" scope="row">Related Concepts:</th>
-						<td class="role-related">
-							<code>&lt;select&gt;</code> in [[HTML]]
-						</td>
+						<td class="role-related"><code>&lt;[^select^]&gt;</code> in HTML</td>
```

Comment:
ReSpec is failing to resolve `[^select^]` here, even though [xref claims to translate it](https://respec.org/xref/?term=select&types=element). @jnurthen, do you know whether I might be doing something wrong here?  Maybe I need to update the xref configuration on [line 146](https://github.com/w3c/aria/blob/06b266f0570c6319c1b5ea4dee316940062ea5ce/index.html#L146)?  If not, do you think it’d be appropriate for me to file a bug at [w3c/respec](https://github.com/w3c/respec/issues)?

The same is true for:

- [`[^input/type/range^]`](https://respec.org/xref/?term=range&types=attr-value)
- [`[^input/value^]`](https://respec.org/xref/?term=value&types=element-attr)

Even though ReSpec is reporting errors, I’ve gone ahead and committed my changes for review for now — I’ll be sure to back these out if there’s no quick resolution.

Reply:
I worked out that this was due to the local data-cite on the sections. This is an override of the document level data-cite. For now I've added html to that but we might want to look at why we have this in the first place (it is to disambiguate the definition of attribute from [dom] and [webidl]. The fact that both these core specs define the same term seems problematic. They should probably use different reference types rather than every spec author having to disambiguate.)

Reply:
I see, thanks for sleuthing that out and explaining.  Where does the document-level configuration for `webidl` live?  I did a little experimentation after your commits to see if I could at least reduce the total number of overrides by favoring `dom html` by default (e.g., [on `<body>`](https://github.com/w3c/respec/wiki/Auto-linking-external-references#:~:text=It%20is%20recommended%20to%20add%20data%2Dcite%20attribute%20on%20the%20%3Cbody%3E.)) and then adding in a lesser number of overrides for `webidl`, but I don’t see where `webidl` is being specified to begin with?

Reply:
I had to chase how this works for webidl and that involved looking at the respec code....

The xref reference in the respecConfig var at the top of the document is where this is normally done BUT webidl doesn't appear there.

It seems that when respec comes across an IDL reference (I didn't look to see what triggers this but guessed it was class="idl") then it traverses back to the closest data-cite and adds webidl to the references. If there is no data-cite then webidl gets added to the document level. 

So - we can resolve this by doing the following:
* Add data-cite="webidl" to the 2 sections of the document that have class="idl"
* fix the 4 other places in the document which are using webidl for references by adding data-cite="webidl" to those places.
We could then remove all instances of data-cite="html" or data-cite="dom" in the document.

I don't know whether this is more or less ugly than the current status.

Reply:
Mmm, thanks, that’s exactly what I had in mind.  I’ll give it a shot and compare its ugliness.  😅

Reply:
In 7d9d11d, I traded 10 `html/dom` overrides for 3 `webidl` ones.  Slightly less ugly?